carbon-design-system/carbon
# fix(DatePicker): remove decorative svg as a click target

Closes #8865 
Closes #9815

The small decorative calendars were click targets. This was causing weird issue with the second calendar opening and setting the first input. Seems to me they should just not receive pointer events at all which seems to address the problem. 

#### Testing / Reviewing

- click the first input and select a date
- observe the first input populate with correct date
- click the second input and select a date
- observe the second input populate with correct date

🏄🏾
